app/controllers/admin/changeSLAction.py

```diff
@@ -0,0 +1,15 @@
+from flask import request, flash
+from app.controllers.admin import admin_bp
+from app.models.course import Course
+from app.models.courseInstructor import CourseInstructor
+from app.models.courseParticipant import CourseParticipant
+from peewee import *
+
+@admin_bp.route('/withdrawCourse/<course>', methods = ['POST'])
+def withdrawCourse(course):
+    course = Course.get(Course.courseName == course)

```

Comment:
I don't think you want this url to use the course name, because then you have to worry about spaces and punctuation in the url

Reply:
Changed it to use courseID instead